.github/workflows/core_crypto_gpu_benchmark.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           sudo apt install ca-certificates curl
           sudo install -m 0755 -d /etc/apt/keyrings
           sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
-          sudo chmod a+r /etc/apt/keyrings/docker.asc  
+          sudo chmod a+r /etc/apt/keyrings/docker.asc
```

Comment:
is that the current recommended way of installing docker on Linux ?

Reply:
It's from the Docker documentation.